docs/content/intro/quickstart.md

```diff
@@ -2,43 +2,265 @@
 title: A Brigade Quickstart
 description: A Brigade Quickstart.
 section: intro
+weight: 22
 ---
 
-## Install Brigade
+In this QuickStart, you will install Brigade, create a project and execute it.
+
+* [Prerequisites](#prerequisites)
+* [Install Brigade](#install-brigade)
+* [Log in to Brigade](#log-in-to-brigade)
+* [Create a Project](#create-a-project)
+* [Trigger an Event](#trigger-an-event)
+
+## Prerequisites
+
+* [A development Kubernetes cluster](#create-a-cluster).
+* [Brigade CLI](#install-the-brigade-cli) installed.
+* [Helm] CLI v3+ installed.
+* [kubectl] CLI installed.
+* Free disk space. The installation requires sufficient free disk space and will fail if your disk is nearly full.
```

Comment:
The homepage of the docs accurately notes that you really don't need to know k8s to use Brigade now. 🎉 

But these prerequisites might _seemingly_ contradict that.

Is it worth mentioning that although Kubernetes acumen, tools, etc. are not required to _use_ Brigade, they are still necessary for _installing_ Brigade?

Reply:
Yeah I was having the same thoughts. I'd taken the verbiage about not needing to know k8s from the v2 proposal and while it's true for _users_ it's not true for _administrators_.